src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- big search bar -->
+    <form
+      action="#"
+      class="search-bar"
+      method="get"
+    >
+      <label class="search-bar__inside">
+        <img 
+          class="search-bar__inside--img big-search-bar__inside-img"
```

Comment:
Looks like you want to use naming by BEM methodology:
but `search-bar__inside--img` - `img` is not modifier for `search-bar__inside`

in this case, the correct hierarchy will be:
`search-bar`
`search-bar__label` 
`search-bar__img`

and then the class with a modifier will look like:
`search-bar__img search-bar__img--small`

Reply:
But you may not use BEM in this task